### PR TITLE
Trafaret for parsing date

### PR DESCRIPTION
**Pull request by vovanbo**

### CODE REVIEW

File: trafaret/contrib/rfc_3339.py [hunk 1]
```diff
@@ -23,3 +23,27 @@ def check_and_return(self, value):
...
+        return "<Date(blank)>" if self.allow_blank else "<Date>"
+
+    def check_and_return(self, value):
+        if isinstance(value, datetime):
+            return value.date()
+        elif isinstance(value, date):
+            return value
+        try:
+            return parse(value).date()
+        except (ValueError, TypeError) as e:
```
Comment (hellysmile) [inline trafaret/contrib/rfc_3339.py]: here can be overflowerror

Comment (Deepwalker) [reply trafaret/contrib/rfc_3339.py]: What value will trigger it?

Comment (hellysmile) [reply trafaret/contrib/rfc_3339.py]: http://dateutil.readthedocs.io/en/stable/parser.html?highlight=overflow

OverflowError – Raised if the parsed date exceeds the largest valid C integer on your system.

Comment (vovanbo) [reply trafaret/contrib/rfc_3339.py]: Yep! I'll fix it tomorrow.

**Comment by vovanbo** [comment]
Is any chance it will be merged soon?

**Review by Deepwalker — APPROVED** [review]
+1

**Pull request merged**